P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+    <p>In the context of dictionaries designed with semantic interoperability in mind, the following
+      example ensures that the <gi>pos</gi> element is interpreted as the same information container
+      as in the case of the example of <tag>f name="POS"</tag> above.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <gramGrp>
+        <pos datcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3"
+          valueDatcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545">NN</pos>
+      </gramGrp>
+    </egXML>
+    <p>Again, the issue of efficiency and redundancy of this type of interoperable markup demands
+      that the references to the particular data categories are provided either in the header of the
+      dictionary or at the level of ODD, in the element <gi>equiv</gi> (child of <gi>elementSpec</gi>), which constitutes an
+      equivalent of the <ident>att.datcat</ident> class.</p>
   </exemplum>
-  <remarks versionDate="2012-04-22" xml:lang="en">
-    <p>ISO 12620:2009 is a standard describing the data model and procedures for a Data
-      Category Registry (DCR). Data categories are defined as elementary descriptors in a
-      linguistic structure. In the DCR data model each data category gets assigned a
-      unique Peristent IDentifier (PID), i.e., an URI. Linguistic resources or preferably
-      their schemas that make use of data categories from a DCR should refer to them using
-      this PID. For XML-based resources, like TEI documents, ISO 12620:2009 normative
-      Annex A gives a small Data Category Reference XML vocabulary (also available online
-      at <ptr target="http://www.isocat.org/12620/"/>),
-      which provides two attributes, <att>dcr:datcat</att> and
-      <att>dcr:valueDatcat</att>.</p>
+  <exemplum xml:lang="en">
+    <p>The <att>targetDatcat</att> attribute is reserved for feature structure declarations, and is
+      analogous to the <att>targetLang</att> attribute of the <ident>att.pointing</ident> class, in that it describes the
+      object that is being referenced, rather than the referencing object.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <fDecl name="POS" targetDatcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3">
+        <fDescr>part of speech (morphosyntactic category)</fDescr>
+        <vRange>
+          <vAlt>
+            <symbol value="NN" datcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545"/>
+            <symbol value="NP" datcat="http://hdl.handle.net/11459/CCR_C-1371_fbebd9ec-a7f4-9a36-d6e9-88ee16b944ae"/>
+            <!-- ... -->
+          </vAlt>
+        </vRange>
+      </fDecl>
+    </egXML>
+    <p>Above, the modeled feature uses <att>targetDatcat</att>, because if it used
+        <att>datcat</att> then we would be saying that the <gi>fDecl</gi> is an instance of the part-of-speech container data category,
```

Comment:
I think this wording is suboptimal, but having a hard time improving it. Perhaps
~~~xml
… because if it used <att>datcat</att> it would be asserting that it is an instance of the part-of-speech container data category, whereas it is not — it models a container (<gi>f</gi>) that encodes a part of speech.
~~~
?

Reply:
Too many "it"s spoil it, I think. And the first of them is my fault. It is not the modeled feature that uses the attribute, but rather the modeling element. Tsk.

Reply:
Changing to 
```xml
 Above, the <gi>fDecl</gi> uses <att>targetDatcat</att>, because if it were to use
        <att>datcat</att>, it would be asserting that it is an instance of the container data
      category <term rend="italic">part of speech</term>, whereas it is not — it models a container (<gi>f</gi>)
      that encodes a part of speech.
```
... while cringing a little bit.